roles/overpass-api/tasks/munin.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: install munin plugins
+  template:
+    src: templates/munin/osm_db_lag
+    dest: /etc/munin/plugins
+    mode: 0755
+
+- name: reload munin-node
+  systemd:
+    name: munin-node
+    state: restarted
```

Comment:
For information, you can use `notify: reload munin-node`, with a file in handlers/munin-node.yml containing something like this:
```
- name: reload munin-node
  service: name=munin-node state=reloaded
```
(should be easily adapted with your systemd syntax)

Reply:
Yes true, same occurs for nginx. Will correct that.